test/debug/test_charptr.py

```diff
@@ -46,7 +47,9 @@ def test_charptr_use_after_implicit_arg_handle_close(compiler, python_subprocess
     result = python_subprocess.run(mod, code)
     assert result.returncode != 0
     assert result.stdout == b""
-    if SUPPORTS_MEM_PROTECTION:
+    if sys.implementation.name == 'pypy':
+        assert b"RPython" in result.stderr
+    elif SUPPORTS_MEM_PROTECTION:
```

Comment:
Out of curiosity: so what does this test give on PyPy?

Reply:
It seems to be different as I change things to support hpy-0.0.4, so this line may need tweeking moving forward.